src/ex1/script.js

```diff
@@ -0,0 +1,82 @@
+let pendingTaskCount = 0;
+
+const taskInput = document.getElementById("todoTaskInput");
+const clearButton = document.getElementById("clearButtonId");
```

Comment:
Very nice 🥇  I love the fact that you initialized them at the beginning. You can call them in all upper case way, to help other devs understand that they are global variables.
like: `TASK_INPUT` and `CLEAR_BUTTON`